### PR TITLE
[Expert] Thermal expertification

### PR DESCRIPTION
**Pull request by MuteTiefling**
Tinker Workbench
image

Vacuumulator
image

Igneous Extruder. Pushing this back a ways so as to promote more creative stone farms earlier on. 
image

Aqueous Accumulator. Similarly pushed slightly back. Hopefully not too far on this. 
image

Nullifier
image

Similarly changed trashcans to be in a similar cost range:
image

Thermal Saw Blade:
image

Sawmill
image

Fluxsaw
image

Fluxbore
image

Noticed IE press wanted Thermal gear press for gears but couldn't accept it. Swapped to IE mold. 

Induction Smelter
image

Environmental Kiln:
image

### CODE REVIEW

**Comment by MuteTiefling** [comment]
Pulverizer
image

Magma Crucible
image

Fluid Encapsulator
image

Blast Chiller
image
Intentionally pushed far back. There are other less obvious ways to farm ice early on if needed. Would rather have people explore those. One being suggested in the recipe itself.

**Comment by MuteTiefling** [comment]
Centrifugal Separator
image

Pyrolyzer. Hopefully not breaking anything with this one. Already required coal coke, so I just expanded that up to blast bricks.
image

Fractionating Still
image

**Comment by MuteTiefling** [comment]
Multiservo Press
image

**Comment by MuteTiefling** [comment]
Insolator pushed back a fair bit
image

Sequential Fabricator as well:
image

Cleaned up recipe input replacement since further changes were made to each of those recipes above.

File: kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/immersivepetroleum/shaped.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+onEvent('recipes', (event) => {
+    if (global.isExpertMode == false) {
+        return;
+    }
+    const id_prefix = 'enigmatica:expert/immersiveengineering/';
```
Comment (NillerMedDild) [inline kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/immersivepetroleum/shaped.js:5]: Typo, immersiveengineering -> immersivepetroleum I think 🙂

Comment (MuteTiefling) [reply kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/immersivepetroleum/shaped.js:5]: Ah good catch

**Comment by MuteTiefling** [comment]
Dynamos!

Stirling
image

Magmatic with light progression into PNC
image

Compression needs a bit more progress into PNC
image

Magical alternatives:
Numismatic requiring a bit of early alchemy

Lapidary with a touch more magical progress
image

**Comment by MuteTiefling** [comment]
Redstone Furnace
image

**Comment by MuteTiefling** [comment]
Think that's about it other than upgrades... which I really need to do more testing to figure out where to put them. Might not move them at all, however, considering the Integral Components are the main gate there.

**Comment by NillerMedDild** [comment]
I think the Stirling Dynamo is a bit on the complex side considering it's the first generator one can make.

Other than that, really nice recipes! I am a bit nervous they're too expensive/micro-crafty, but we'll have to gauge that in playtesting.

**Comment by MuteTiefling** [comment]
Hrm, drop the andesite alloy from stirling maybe?

**Comment by NillerMedDild** [comment]
Yeah that ought to do the trick :)

**Comment by MuteTiefling** [comment]
Alright, simplified it back to more vanilla materials:
image

**Review by NillerMedDild — APPROVED** [review]

**Comment by NillerMedDild** [comment]
Nice! Thank you :D

**Pull request merged**